azurerm/internal/services/iothub/client/client.go

```diff
@@ -1,6 +1,7 @@
 package client
 
 import (
+	"github.com/Azure/azure-sdk-for-go/services/digitaltwins/mgmt/2020-10-31/digitaltwins"
```

Comment:
Actually why was this added to the iothub service? this is a new package import so shouldn't it be a new service package??